f5_openstack_agent/lbaasv2/drivers/bigip/tenants.py

```diff
@@ -81,13 +84,30 @@ def assure_tenant_created(self, service):
                         self.network_helper.create_route_domain(
                             bigip,
                             rd_id,
+                            service['qos'],
                             folder_name,
                             self.conf.f5_route_domain_strictness)
                     except Exception as err:
                         LOG.exception(err.message)
                         raise f5ex.RouteDomainCreationException(
                             "Failed to create route domain for "
                             "tenant in %s" % (folder_name))
+                else:
+                    LOG.debug("Try to update route domain")
+                    try:
+                        payload = self.network_helper.route_domain_update
+                        payload['name'] = folder_name
+                        payload['partition'] = folder_name
+                        if service['qos'] == '':
+                            payload['bwcPolicy'] = 'None'
```

Comment:
Not sure why we need to use 'None', instead of None. Any purpose?

Reply:
Qin, the "None" is the fixed none at bigip side which means no any bwc control profile is bound. 
We had better keep this as "None" in order to keep the consistence between openstack and bigip.